app/src/main/java/com/io/moviO/MoviesListFragment.kt

```diff
@@ -5,17 +5,26 @@ import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProvider
 import com.io.moviO.adapter.MovieListAdapter
 import com.io.moviO.databinding.FragmentMoviesListBinding
 
-class MoviesListFragment : Fragment(R.layout.fragment_movies_list), MovieListAdapter.OnMovieClickedListener {
+class MoviesListFragment : Fragment(R.layout.fragment_movies_list), MovieListAdapter.OnMovieClickedListener, LifecycleOwner {
```

Comment:
why do you implement this -> LifecycleOwner? Isn't the Fragment implementing it by default?

Reply:
You are right. I checked it